styles/globals.css

```diff
@@ -40,3 +40,19 @@ input[type="search"]::-webkit-search-cancel-button {
 .z-50 {
   z-index: 50;
 }
+
+.-left-17 {
+  left: -4.4rem;
+}
+
+.custom-select__select-item[data-highlighted] {
```

Comment:
please remove these lines and use the already existing https://github.com/ecklf/tailwindcss-radix plugin to set these classes. At the moment the plugin is configured with the default options, so reading the link should answer most questions, however, if you need advanced configuration, let us know and we can pair on this. Since the plugin is covering all the possible state classes, this should not be needed at all 🙉 🙈

Reply:
@0-vortex we don't have that plugin in this repo yet, I'm going to install it and test it out.

Reply:
Oh really sorry about that too, thought we ported this over from hot.opensauced but apparently we didn't; that could explain some of the radix inconsistencies we were seeing that were generally magically solved by that plugin 😅

Reply:
No problem :+1: